dask_kubernetes/operator/controller/controller.py

```diff
@@ -260,6 +261,9 @@ async def startup(settings: kopf.OperatorSettings, **kwargs):
     # The default timeout is 300s which is usually to long
     # https://kopf.readthedocs.io/en/latest/configuration/#networking-timeouts
     settings.networking.request_timeout = 10
+    global SIZE, DELAY
+    SIZE = int(os.getenv("WORKER_BATCH_SIZE"))
+    DELAY = int(os.getenv("WORKER_DELAY")[:-1])
```

Comment:
It would be nice if we could use the Dask config system instead as that supports environment variables.